astropy/uncertainty/builtin_distrs.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""
+Built-in distribution-creation functions.
+"""
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import numpy as np
+
+from ..units import UnitsError
+from .core import Distribution
+
+__all__ = ['normal', 'poisson', 'uniform', 'uniform_center_width']
+
+
+def normal(center, std=None, var=None, ivar=None, n_samples=1000,
+           dcls=Distribution, **kwargs):
+    """
+    Create a Gaussian/normal distribution.
+
+    Parameters
+    ----------
+    center : `~astropy.units.Quantity`
+        The center of this distribution
+    std : `~astropy.units.Quantity` or `None`
+        The standard deviation/σ of this distribution. Shape must match and unit
+        must be compatible with ``center``, or be `None` (if ``var`` or ``ivar``
+        are set).
+    var : `~astropy.units.Quantity` or `None`
```

Comment:
Because it's a convenience function, and lots of users prefer to carry around ivar instead of std, for example. I agree it's rather trivial, but given that it's already a "convenience" function I prefer to make it extra-convenient.